### ISSUE TITLE
Welcome: NVDA does not read the link's context clearly for "more" and "others" links.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51978]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 2.4.4
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Navigate to Welcome screen.
3. Read the links "more" and "others" present under headings "Tools and Languages" and "Settings and Keybindings" respectively.

**Actual:**
NVDA does not read the link's context clearly for "more" and "others" links. These links does not convey proper information to the user what does these links stand for and where the user will be redirected to. 

**Same issue has been observed with JAWS18.**

**Expected:**
NVDA should read the link's context clearly as - "Install additional support for more languages" and "Install settings and keyboard shortcuts for others" for "more" and "others" languages respectively.

**Recommendations:**
Add aria-label with detailed description about the purpose of the links.
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not be able to understand the context of these links if proper description is not given.  

**MAS Reference:**
MAS 2.4.4 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}

**Attachment For Reference:**
[A11y_VSCode_WelcomeScreen_ScreenReader_LinkContext.pptx](https://github.com/Microsoft/vscode/files/2105290/A11y_VSCode_WelcomeScreen_ScreenReader_LinkContext.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
I feel like this is an easy aria label change, assigning to July
